src/Microsoft.CodeAnalysis.ResxSourceGenerator/Microsoft.CodeAnalysis.ResxSourceGenerator.UnitTests/Verifiers/CSharpSourceGeneratorVerifier`1.cs

```diff
@@ -1,9 +1,11 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
 
+#nullable enable
+
 namespace Microsoft.CodeAnalysis.ResxSourceGenerator.Test
 {
     public static partial class CSharpSourceGeneratorVerifier<TSourceGenerator>
-        where TSourceGenerator : ISourceGenerator, new()
+        where TSourceGenerator : IIncrementalGenerator, new()
```

Comment:
just to check the intent here.  this will make it so you can only test new-style incremnetal generator, not legacy ISGs?  yes?

Reply:
Yes, but only inside this one test project. There are not `ISourceGenerator` implementations under test by this project, so the old behavior is no longer needed.

Reply:
gotcha.  that seems good then :)